experimental/devices/pca9685/pca9685.go

```diff
@@ -0,0 +1,150 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package pca9685
+
+import (
+	"time"
+
+	"periph.io/x/periph/conn/gpio"
+	"periph.io/x/periph/conn/i2c"
+	"periph.io/x/periph/conn/physic"
+)
+
+// I2CAddr i2c default address.
+const I2CAddr uint16 = 0x40
+
+// PCA9685 Commands
+const (
+	mode1      byte = 0x00
+	mode2      byte = 0x01
+	subAdr1    byte = 0x02
+	subAdr2    byte = 0x03
+	subAdr3    byte = 0x04
+	prescale   byte = 0xFE
+	led0OnL    byte = 0x06
+	led0OnH    byte = 0x07
+	led0OffL   byte = 0x08
+	led0OffH   byte = 0x09
+	allLedOnL  byte = 0xFA
+	allLedOnH  byte = 0xFB
+	allLedOffL byte = 0xFC
+	allLedOffH byte = 0xFD
+
+	// Bits
```

Comment:
In a follow up, I think it's worth describing what they are for. This is mixing bitmasks for mode2 (ourDrv, invrt), mode1 (allCall, sleep) and restart is unused and I couldn't find it in the datasheet.

So remove restart for now, we'll handle the rest later.